### PR TITLE
Cran update

### PR DESCRIPTION
**Pull request by hongooi73**
### Description

Updates the R code in the examples for the latest tidyverts package versions on CRAN.

### Checklist:

- [x] My code follows the code style of this project, as detailed in our [contribution guidelines](../CONTRIBUTING.md).
- [ ] I have added tests.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: examples/retail_turnover/README.md [hunk 1]
```diff
@@ -23,22 +23,11 @@ The following packages and their dependencies are needed to run the notebooks in
 | Future    | future, future.apply |
 | Other     | urca, rmarkdown, distributional, devtools (see below) |
```
Comment (chenhuims) [inline examples/retail_turnover/README.md:24]: It seems that you don't install devtools anymore. Do you need to keep `devtools` here?

File: examples/retail_turnover/README.md [hunk 2]
```diff
@@ -23,22 +23,11 @@ The following packages and their dependencies are needed to run the notebooks in
...
-# install Tidyverts packages from GitHub
-install.packages("devtools")
-
-devtools::install_github("tidyverts/tsibble@a19cda281c3f1e0061b5b0de93b059c52052ebda")
-devtools::install_github("tidyverts/tsibbledata@b06a965b788722157a149296c47f821c99cc41f0")
-devtools::install_github("mitchelloharawild/distributional@e668b520b415f417f71eacd7e1e940561eecffd6")
-devtools::install_github("tidyverts/fabletools@864b2daa983446017f2ed757a3b8889b935cc2cb")
-devtools::install_github("tidyverts/fable@d2600c151fde1609cc491d8f94bd136c71f87523")
-devtools::install_github("tidyverts/feasts@f006746effa10bc223479441ebede136ca016b11")
+install.packages(c("tsibble", "tsibbledata", "fabletools", "fable", "feasts"))
```
Comment (chenhuims) [inline examples/retail_turnover/README.md:30]: Should you also install these packages that you mentioned above - `dplyr`, `tidyr`, `ggplot2`, `distributional`?

File: examples/retail_turnover/README.md (hunk 1 above)
Comment (hongooi73) [reply examples/retail_turnover/README.md]: Fixed

File: examples/retail_turnover/README.md (hunk 2 above)
Comment (hongooi73) [reply examples/retail_turnover/README.md:30]: That's fine, they are dependencies and will be installed automatically. I should remove distributional from the list though.

Comment (chenhuims) [reply examples/retail_turnover/README.md:30]: Sounds good.

**Pull request merged**